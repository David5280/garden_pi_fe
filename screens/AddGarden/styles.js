import { StyleSheet, Dimensions } from 'react-native';

export default StyleSheet.create({
  screenContainer: {
    alignItems: 'center',
    height: Dimensions.get('window').height,
    paddingTop: 50,
    width: Dimensions.get('window').width,
  },
  infoContainer: {
    alignItems: 'center',
    backgroundColor: 'rgba(115, 166, 134, .7)',
    padding: 10,
    margin: 5,
    width: Dimensions.get('window').width * .95,
    borderColor: '#A14550',
    borderRadius: 30,
    borderWidth: 2,
    shadowColor: '#000',
    shadowOffset: { width: 2, height: 2 },
    shadowOpacity: 1,
  },
  headerContainer: {
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'center',
    width: Dimensions.get('window').width * .8,
  },
  header: {
    paddingTop: 200
  },
  backBtn: {
    width: 35,
    height: 35
  },
})