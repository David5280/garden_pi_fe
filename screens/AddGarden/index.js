import React, { Component } from 'react';
import { 
  View, 
  Text, 
  TouchableOpacity, 
  Button, 
  Dimensions, 
  ImageBackground,
  Image } from 'react-native';
import styles from './styles';

export class AddGarden extends Component {
  constructor() {
    super()
  }

  render() {
    return (
      <View>
      <ImageBackground
        source={require('../../assets/images/pottedPlants.jpg')}
        style={styles.screenContainer}
        onLoad={this.toggleBackgroundLoaded}
      >

      </ImageBackground>
      </View>
    )
  }
}

export default AddGarden;