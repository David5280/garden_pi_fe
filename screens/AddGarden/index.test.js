import React from 'react';
import AddGarden from './index';
import 'react-native';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<AddGarden />).toJSON();
  expect(tree).toMatchSnapshot();
});