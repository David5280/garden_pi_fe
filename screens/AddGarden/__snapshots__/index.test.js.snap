// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "alignItems": "center",
        "height": 1334,
        "paddingTop": 50,
        "width": 750,
      }
    }
  >
    <Image
      source=""
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": 1334,
            "width": 750,
          },
          undefined,
        ]
      }
    />
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(115, 166, 134, .7)",
            "borderColor": "#A14550",
            "borderRadius": 30,
            "borderWidth": 2,
            "margin": 5,
            "padding": 10,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 2,
              "width": 2,
            },
            "shadowOpacity": 1,
            "width": 712.5,
          },
          Object {
            "height": 106.72,
          },
        ]
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
            "width": 600,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <Image
            source=""
            style={
              Array [
                Object {
                  "height": 35,
                  "width": 35,
                },
                Object {
                  "marginRight": 10,
                },
              ]
            }
          />
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "lemondrop",
                  "textShadowColor": "black",
                  "textShadowOffset": Object {
                    "height": 2,
                    "width": 2,
                  },
                  "textShadowRadius": 5,
                },
                Object {
                  "color": "#E64C3C",
                  "fontSize": 35,
                },
              ]
            }
          >
            G
          </Text>
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "lemondrop",
                  "textShadowColor": "black",
                  "textShadowOffset": Object {
                    "height": 2,
                    "width": 2,
                  },
                  "textShadowRadius": 5,
                },
                Object {
                  "color": "#FF9933",
                  "fontSize": 35,
                },
              ]
            }
          >
            A
          </Text>
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "lemondrop",
                  "textShadowColor": "black",
                  "textShadowOffset": Object {
                    "height": 2,
                    "width": 2,
                  },
                  "textShadowRadius": 5,
                },
                Object {
                  "color": "#F1C431",
                  "fontSize": 35,
                },
              ]
            }
          >
            R
          </Text>
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "lemondrop",
                  "textShadowColor": "black",
                  "textShadowOffset": Object {
                    "height": 2,
                    "width": 2,
                  },
                  "textShadowRadius": 5,
                },
                Object {
                  "color": "#187F03",
                  "fontSize": 35,
                },
              ]
            }
          >
            D
          </Text>
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "lemondrop",
                  "textShadowColor": "black",
                  "textShadowOffset": Object {
                    "height": 2,
                    "width": 2,
                  },
                  "textShadowRadius": 5,
                },
                Object {
                  "color": "#3998DB",
                  "fontSize": 35,
                },
              ]
            }
          >
            E
          </Text>
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "lemondrop",
                  "textShadowColor": "black",
                  "textShadowOffset": Object {
                    "height": 2,
                    "width": 2,
                  },
                  "textShadowRadius": 5,
                },
                Object {
                  "color": "#4b0082",
                  "fontSize": 35,
                },
              ]
            }
          >
            N
          </Text>
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "lemondrop",
                  "textShadowColor": "black",
                  "textShadowOffset": Object {
                    "height": 2,
                    "width": 2,
                  },
                  "textShadowRadius": 5,
                },
                Object {
                  "color": "#7f00ff",
                  "fontSize": 35,
                },
              ]
            }
          >
            Pi
          </Text>
        </View>
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <Image
            source=""
            style={
              Array [
                Object {
                  "height": 35,
                  "width": 35,
                },
                Object {
                  "marginLeft": 10,
                },
              ]
            }
          />
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(115, 166, 134, .7)",
          "borderColor": "#A14550",
          "borderRadius": 30,
          "borderWidth": 2,
          "margin": 5,
          "padding": 10,
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 2,
            "width": 2,
          },
          "shadowOpacity": 1,
          "width": 712.5,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#d5fdd5",
            "fontFamily": "Helvetica",
            "fontSize": 16,
            "fontWeight": "bold",
            "marginBottom": 2,
            "textAlign": "center",
            "textShadowColor": "black",
            "textShadowOffset": Object {
              "height": 2,
              "width": 2,
            },
            "textShadowRadius": 5,
          }
        }
      >
        Add a New Garden
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCapitalize="words"
        autoCorrect={true}
        onChangeText={[Function]}
        placeholder="Garden Name"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#d5fdd5",
            "borderColor": "black",
            "borderRadius": 30,
            "borderWidth": 2,
            "color": "black",
            "height": 50,
            "paddingLeft": 20,
            "width": 562.5,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
      <TextInput
        allowFontScaling={true}
        keyboardType="decimal-pad"
        onChangeText={[Function]}
        placeholder="Max Moisture (0-100)"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#d5fdd5",
            "borderColor": "black",
            "borderRadius": 30,
            "borderWidth": 2,
            "color": "black",
            "height": 50,
            "paddingLeft": 20,
            "width": 562.5,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
      <TextInput
        allowFontScaling={true}
        keyboardType="decimal-pad"
        onChangeText={[Function]}
        placeholder="Min Moisture (0-100)"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#d5fdd5",
            "borderColor": "black",
            "borderRadius": 30,
            "borderWidth": 2,
            "color": "black",
            "height": 50,
            "paddingLeft": 20,
            "width": 562.5,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
      <Text>
        Location: 
      </Text>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#FAF0E6",
            "borderColor": "#A14550",
            "borderRadius": 10,
            "borderWidth": 2,
            "height": 75,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 12,
            "marginRight": 12,
            "marginTop": 5,
            "opacity": 1,
            "overflow": "hidden",
            "width": 300,
          }
        }
      >
        <Text>
          Submit
        </Text>
      </View>
    </View>
  </View>
</View>
`;
